igamt-lite-client/app/views/edit/editMessages.html

```diff
@@ -625,21 +625,37 @@
             </a>
             <ng-form class="form-inline">
                 <span class="label label-success label-icon">G</span>
+                <span ng-show="editableGrp!==node.obj.id" context-menu="segOption" ng-bind="getGroupNodeName(node.obj)" />
                 <span ng-show="editableGrp===node.obj.id">
-                    <select ng-show="editableGrp===node.obj.id" ng-model="position" ng-change="selectPos()"  style="width:15%" class="form-control input-sm">
```

Comment:
The ng-show is missing in the material version. Is that normal ? or by mistake ?